presto-raptor/src/test/java/com/facebook/presto/raptor/integration/TestRaptorIntegrationSmokeTest.java

```diff
@@ -838,6 +846,12 @@ public void testAlterTable()
         assertUpdate("DROP TABLE test_alter_table");
     }
 
+    @Test
+    public void testAlterTableUnsupportedType() {
```

Comment:
nit: Move the left brace to the next line, otherwise the check style will complain :)